advisories/hackage/cryptonite/HSEC-2025-0002.md

```diff
@@ -0,0 +1,41 @@
+```toml
+[advisory]
+id = "HSEC-2025-0002"
+cwe = []
+keywords = ["crypto", "historical"]
+related = ["GHSA-w5vr-6qhr-36cc"]
+
+[[affected]]
+package = "cryptonite"
```

Comment:
It looks like this also affecting crypton (https://github.com/kazu-yamamoto/crypton/blob/main/Crypto/PubKey/Ed25519.hs#L104)

Reply:
Ouch, they share the same code:

* https://github.com/haskell-crypto/cryptonite/blob/cryptonite-v0.30/cbits/ed25519/ed25519.c
* https://github.com/kazu-yamamoto/crypton/blob/48fb9df2de5ee752196724b081f4d3cdb57576ed/cbits/ed25519/ed25519.c#L53

I've added crypton